docs/changelog.md

```diff
@@ -236,6 +237,18 @@ ElasticSearch 8 doesn't allow creating new indexes with `Mapping.PREFIX_TREE` ma
 using `Mapping.PREFIX_TREE` will work in ElasticSearch 8 after migration. See 
 [ElasticSearch 8 migration guide](https://www.elastic.co/guide/en/elasticsearch/reference/current/migrating-8.0.html#geo-shape-strategy).
 
+##### Add support for ScyllaDB driver
```

Comment:
Not that important, but I would move this section and the one above about ES 8 at least above the deprecation notices. Those deprecations should be the last part of the changelog and I would order the changelog in general by priority. Maybe move both sections right behind the breaking changes?

Reply:
Moving both sections above the deprecation notices, but we probably need to reorder that list before the release.